src/main/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClient.java

```diff
@@ -24,20 +24,22 @@
 
 public class PDFServiceClient {
 
-    private static final String GENERATE_FROM_HTML_ENDPOINT_PATH = "/html";
     private static final Logger LOGGER = LoggerFactory.getLogger(PDFServiceClient.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
does this need to be static?

Reply:
I hardly see any reason *not* to have it static. It is not set by client programmer, nor modified from the outside, it's pretty much constant across all object instances.

Reply:
if it's thread safe, fair enough